apps/api/package.json

```diff
@@ -47,7 +49,8 @@
     "scripts:populate-object-index": "tsx src/scripts/populateObjectIndex.ts",
     "generate-openapi": "tsx -e \"const { swaggerSpec } = require('./src/config/swagger'); require('fs').writeFileSync('./openapi/openapi.json', JSON.stringify(swaggerSpec, null, 2));\"",
     "generate-markdown": "openapi-markdown -i ./openapi/openapi.json -o ./openapi/API.md",
-    "generate-docs": "pnpm generate-openapi && pnpm generate-markdown"
+    "generate-docs": "pnpm generate-openapi && pnpm generate-markdown",
+    "generate": "pnpm generate-docs && pnpm format"
```

Comment:
added `pnpm generate` so that we can ensure OpenAPI spec changes are also formatted by default